### PR TITLE
Add Invocation Params to onLLMStart

### PR DESCRIPTION
**Pull request by vowelparrot**

### CODE REVIEW

File: langchain/src/chat_models/base.ts [hunk 1]
```diff
@@ -72,9 +73,13 @@ export abstract class BaseChatModel extends BaseLanguageModel {
       this.callbacks,
       { verbose: this.verbose }
     );
+    const invocationParams = { invocation_params: this?.invocationParams() };
     const runManager = await callbackManager_?.handleChatModelStart(
       { name: this._llmType() },
-      messages
+      messages,
+      uuidv4(),
```
Comment (nfcampos) [inline langchain/src/chat_models/base.ts:80]: This should be undefined

File: langchain/src/llms/base.ts [hunk 2]
```diff
@@ -88,10 +96,15 @@ export abstract class BaseLLM extends BaseLanguageModel {
       this.callbacks,
       { verbose: this.verbose }
     );
+    const invocationParams = { invocation_params: this?.invocationParams() };
     const runManager = await callbackManager_?.handleLLMStart(
       { name: this._llmType() },
-      prompts
+      prompts,
+      uuidv4(),
```
Comment (nfcampos) [inline langchain/src/llms/base.ts:103]: This should be undefined

**Review by nfcampos — COMMENTED** [review]
Looks good, two comments

File: langchain/src/llms/base.ts (hunk 2 above)
Comment (vowelparrot) [reply langchain/src/llms/base.ts:103]: Makes sense! TY! I'm assuming that means explicitly passing `undefined`

**Pull request merged**